src/archivematicaCommon/lib/elasticSearchFunctions.py

```diff
@@ -991,3 +1069,20 @@ def normalize_results_dict(d):
     This normalizes the dict by replacing the arrays with their first value.
     """
     return {k: v[0] for k, v in d['fields'].iteritems()}
+
+def augment_raw_search_results(raw_results):
+    """
+    This function takes JSON returned by an ES query and removes some superfluous data from the structure.
```

Comment:
Maybe be more specific and say that it returns the source document for each result?